### PR TITLE
Custom Connector Implementation

### PR DESCRIPTION
**Pull request by SnehaSuresh-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-2187

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/CustomEmailConnector.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+import java.util.List;
+import java.util.logging.Logger;
+
+
+public class CustomEmailConnector extends AbstractConnector<ConnectorRequest, EmptyResponse> {
+
+    private final Logger LOGGER = Logger.getLogger(CustomEmailConnector.class.getName());
+
+    public static final String CONNECTOR_ID = "camunda-email-connector";
```
Comment (sumesh-aot) [inline forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/CustomEmailConnector.java:20]: Can we make the connector id generic ?

File: forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/response/EmptyResponse.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (sumesh-aot) [inline forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/response/EmptyResponse.java:1]: Can you move this out of mail package, as this class would be useful for other connectors too

File: forms-flow-bpm/forms-flow-bpm-integration/src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,10 @@
+spring.application.name=Formsflow Integration
+
+
+# E-mail configurations
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=sneha.suresh@aot-technologies.com
+spring.mail.password=mscvohwtkkbqggzm
```
Comment (sumesh-aot) [inline forms-flow-bpm/forms-flow-bpm-integration/src/main/resources/application.properties:8]: please remove the password and username.
Also can we move this to yaml as we follow yaml in bpm for properies.
And shouldn't this be on the main package than the sub-module ?

**Review by sumesh-aot — COMMENTED** [review]
Please add test cases too -> Mock java mail

File: forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/CustomEmailConnector.java (hunk 1 above)
Comment (SnehaSuresh-aot) [reply forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/CustomEmailConnector.java:20]: This connector id is specific to the mail connector. I think we can make this as a common variable constants

**Pull request closed without merging**